lib/drivers/direct/container.js

```diff
@@ -101,6 +101,13 @@ Container.prototype.setStartOptions = function(options) {
       self.debug('set-size %j: rsp %j', options.size, rsp);
     });
   }
+
```

Comment:
I'm OK to LGTM this for now, it unblocks QA, but we need to modify mesh-models to only update if there are changes.

Also, this code will need to be copied into the docker driver.